contribution-guide.md

```diff
@@ -0,0 +1,157 @@
+    - [GitHub][(https://github.com/ballerina-platform/ballerina-lang/issues): to file issues, comment on other issues, send your pull requests.
+
+- **IMPORTANT:** Send an email to security@ballerina.io to report sensitive security-related issues. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+
+- Submitting a bug is just as important as contributing to code. Create a new issue in the relevant GitHub repo using the below links, fill out all sections in the issue template, and submit the bug/issue report. 
+
+    >**Tip:** If you are unsure whether you have found a bug, search existing issues in the corresponding repo on GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
+   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
+
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
+
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+3. Download the Java SE Development Kit (JDK) version 8 from one of the following locations and install it.
+    - [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html) 
+>**Note:** Set the `JAVA_HOME` environment variable to the path name of the directory into which you installed JDK.
+4. Download [Node.js](https://nodejs.org/en/download/) (version 8.9.x or the latest LTS release) and install it.
+5. Download [npm](https://www.npmjs.com/get-npm) (version 5.6.0 or later) and install it.
+
+## Obtain the source code 
+
+1. Execute the below command to clone the ballerina-lang Git repo.
+```bash 
+git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
+```
+>**Tip:** If you have already forked the repository to your GitHub account, then execute the below command replacing 'YOUR-GITHUB-USERNAME' with your Git username.
+```bash 
+git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
+```
+2. Execute the below command to update the Git submodules.
+```bash 
+git submodule update --init
+```
+## Build the project
+
+1. Navigate to the <BALLERINA-LANG_REPO> root directory and execute the below Gradle command.
```

Comment:
What it is meant by <BALLERINA-LANG_REPO> is not clear here. Shall we simply say Ballerina root directory?

Reply:
Changed to "BALLERINA_PROJECT_ROOT" by introducing it as follows in its first occurrence:

Navigate to the root directory of the Ballerina project (i.e., <BALLERINA_PROJECT_ROOT>) and execute the below Gradle command.

Reply:
Change all the occurrences of <BALLERINA_LANG_REPO> with <BALLERINA_PROJECT_ROOT>

Reply:
Fixed all occurrences.